presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java

```diff
@@ -15,12 +16,18 @@
 public class ImagesPresenter {
 
     private ImagesView view;
-    private GetLatestImagesUseCase getLatestImagesUseCase;
+//    private GetLatestImagesUseCase getLatestImagesUseCase;
```

Comment:
remove comment if you dont  need it

Reply:
The commented code is there because it will be used in the next stages. It is also part of what was in the initial project when i download it. I commented  it so as not to get confused because I was not going to use it now.

Reply:
ok np